File: luhn.js
let numbers = prompt('Enter the Numbers: ');
numbers = numbers.split('');
let count = 0;
for (i = 0; i < numbers.length; i++) {
    let number = Number(numbers[i]);
    if (i % 2 != 0) {
        let num = (number * 2).toString();
        if (num.length == 2) {
            count += Number(num[0]) + Number(num[1]);
        } else {
            count += number
        }
    } else {
        count += number
    }
}

let roundOf = Math.ceil(count/10) * 10;
let x = roundOf - count;
console.log(count)
console.log(x)

File: luhn.py
numbers = input('Enter the Numbers: ')
if not numbers.isdigit() or len(numbers) != 14:
    print('Input is not a number or does not have exactly 14 digits')
else:
    count = 0
    for i in range(len(numbers)):
        number = int(numbers[i])
        if i % 2 != 0:
            num = str(number * 2)
            if len(num) == 2:
                count += int(num[0]) + int(num[1])
            else:
                count += number
        else:
            count += number

    round_of = ((count + 9) // 10) * 10
    x = round_of - count
    print(count)
    print(x)
